data/data-pipeline/data_pipeline/tests/score/test_calculation.py

```diff
@@ -113,6 +218,27 @@ def test_percentile_columns(final_score_df):
         low_high_school,
         donut_hole_income,
         donut_hole_adjacency,
+        dot_burden,
+        diesel,
+        traffic_proximity,
+        energy_burden,
+        pm25,
+        kitchen_plumbing,
+        housing,
+        non_natural_space,
+        rmp,
+        npl,
+        tsdf,
+        wastewater,
+        ust,
+        diabetes,
+        asthma,
+        heart_disease,
+        low_life_expectancy,
+        unemployment,
+        low_median_income,
+        linguist_isolation,
+        poverty,
```

Comment:
and then, just for my own accounting, we have the three boolean factors (aml, hrs, and fuds) and the composite fancy lead indicator. this is great!